Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleSceneView.swift

```diff
@@ -296,6 +313,7 @@ public extension WorldScaleSceneView {
     }
 }
 
+#if !os(visionOS)
```

Comment:
Could the enclosing function be annotated as unavailable on visionOS?

Reply:
Same response as [here](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/875/files#r1768811353).